scss/base/_typography.scss

```diff
@@ -266,6 +272,8 @@ body {
   --body-line-height: #{$body-line-height--L1};
   --body-paragraph: #{$body-paragraph--L1};
 
+  --body-font-weight--emphasized: #{$body-font-weight--emphasized--L1};
```

Comment:
notice --header font weight was using the L1 variable, so using the L1 for body as well